chromium_src/content/browser/permissions/permission_controller_impl.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "content/public/browser/permission_type.h"
+
+#define BRAVE_PERMISSION_TYPES \
+    case PermissionType::AUTOPLAY:
+#include "../../../../../content/browser/permissions/permission_controller_impl.cc"  // NOLINT
+#undef NUM
```

Comment:
?

Reply:
this wouldn't even do anything here anyway so can you remove in a followup to avoid confusion?

Reply:
Wrong #undef notwithstanding, without this I get the following compilation error:
<pre>
brave-browser\src\brave\chromium_src\content\browser\permissions/../../../../../content/browser/permissions/permission_controller_impl.cc(35,11):
error: enume ration value 'AUTOPLAY' not handled in switch [-Werror,-Wswitch]
switch (permission_name) {
^
brave-browser\src\brave\chromium_src\content\browser\permissions/../../../../../content/browser/permissions/permission_controller_impl.cc(72,1):
error: contro l may reach end of non-void function [-Werror,-Wreturn-type]
}
^
2 errors generated.
</pre>

Reply:
Wrong undef addressed in 99847be